target/linux/ramips/dts/mt7628an_rakwireless_rak634.dts

```diff
@@ -17,6 +17,16 @@
 			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
 		};
 	};
+	
+	sdhci@10130000 {
```

Comment:
Why is this commit separate from the device support one?

Reply:
Because I develop the support for this device in to stages:
1) All basic things is working
2) SD card support

Reply:
Why open a PR if its not complete?